src/pages/patientView/genomicOverview/tracksHelper.ts

```diff
@@ -132,16 +132,21 @@ export function getChmInfo(genomeBuild:string) {
     return sel;
 }
 
-export function plotChromosomes(p: any, config: any,chmInfo: any) {
+export function plotChromosomes(p: any, config: any,chmInfo: any, genomeBuild: any) {
     var yRuler = config.rowMargin+config.ticHeight;
     drawLine(config.wideLeftText,yRuler,config.wideLeftText+config.GenomeWidth,yRuler,p,'#000',1);
     // ticks & texts
     for (var i=1; i<chmInfo.genomeRef.length; i++) {
+        if (chmInfo.genomeRef[i] === 0 && genomeBuild === "GRCm38") {
+            // Do not display chromosomes 20, 21 and 22 in mouse, they do not exist
```

Comment:
nice comment!

Reply:
on further thought this seems like a cloodge.     DOes the 0 value indicate more generally that chromosome doesn't exist?   Shouldn't the collection of chromosome data reflect the species chromosome further upstream?   At very least, lets make a constant for the genomeBuild value . MOUSE_GENOME or something like that.

Reply:
@alisman Because this was created for human, chromosomes 23 and 24 are turned to X and Y. I thought the best way to adapt it for mouse without touching the code too much was to set the length of chromosomes 20, 21 and 22, which do not exist in mouse, to 0, to respect the structure of `chromosomeSizes.json`.

Regarding the genome build names, they are unique, even between species. For example, build 38 in human is `GRCh38`, while the mouse one is `GRCm38`.